Makefile

```diff
@@ -459,7 +459,7 @@ catalog-push: ## Push a catalog image.
 # Deploy the catalog.
 .PHONY: catalog-deploy
 catalog-deploy: ## Deploy a catalog image.
-	$(SED) -e 's~<IMAGE>~$(CATALOG_IMAGE)~' ./config/samples/catalog/catalog.yaml | kubectl apply -f -
+	$(SED) -e 's~<IMAGE>~$(CATALOG_IMAGE)~' ./config/samples/catalog/catalog.yaml | kubectl apply --server-side -f -
```

Comment:
I don't think it's needed on the catalog here. Only CRD is affected by the issue